x11-apps/copyq/ChangeLog

```diff
@@ -0,0 +1,70 @@
+# ChangeLog for x11-apps/copyq
```

Comment:
what's that? we do not ship separate ChangeLog files since 2015

Reply:
Ok sorry, not sure how that got in there, I'll remove it